docs/manual/nullness-checker.tex

```diff
@@ -1251,6 +1251,18 @@
 To use the \refclass{checker/nullness/util}{Opt} class, the
 \<checker-util.jar> file must be on the classpath at run time.
 
+It is true that \<Optional> can express types that cannot be expressed via
```

Comment:
Kenn, thanks for his addition.  I would like to include information to address this issue, but I would like to clarify the description first.

Could you please include an example?  The current description is a bit vague, making it harder to understand than necessary.  It also uses undefined terms like "composed"; please define that, or replace it, or give an example.

For a command-line parameter, an invalid or default value can often represent a missing argument.  (For example, `""` for a string or -1 for an Integer.)  You might argue that is bad style, or make an argument that it is not always possible, but please make such an argument explicitly, or else the writing will not be as persuasive as it should be.

The compositionality advocated by this patch violates the [first rule about `Optional`](https://checkerframework.org/manual/#optional-checker): "Never, ever, use null for an Optional variable or return value.".  It may be of value to sometimes violate a rule, but please do address this explicitly and justify exactly when and why you think the rule should be violated.

I wonder if this text belongs in Section 3.7.3  "Relationship to `Optional<T>`" or Chapter 5 "Optional Checker for possibly-present data" rather than at this location.  There could be a cross-reference to it from the current location, of course.

Finally, CI is failing.  Could you please add your name to `docs/manual/contributors.tex` as described in the [CI failure message](https://dev.azure.com/checkerframework/checkerframework/_build/results?buildId=8145&view=logs&jobId=4b19ccca-7d7d-5749-26de-b87349fba2b4&j=4b19ccca-7d7d-5749-26de-b87349fba2b4&t=4dba1f6c-8e7b-5172-04db-4dd554c55b12)?

Thanks again!